scripts/compiling.py

```diff
@@ -0,0 +1,322 @@
+import lxml.etree as ET
+import sys
+import os
+
+# shorthand for yaml tabulation
+def getTabs(n):
```

Comment:
Move this functions to `utils.py` or similar

Reply:
https://github.com/erlerobot/HRIM/commit/72387750978665ba428c3c4214723ae2957a7cd1